src/librustc/session/config.rs

```diff
@@ -1387,6 +1387,10 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
           "output a json file with profiler results"),
     emit_stack_sizes: bool = (false, parse_bool, [UNTRACKED],
           "emits a section containing stack size metadata"),
+    plt: Option<bool> = (Some(false), parse_opt_bool, [TRACKED],
```

Comment:
Shouldn't this be initialized `None`, so we know whether it has been explicitly set?

Reply:
Yes, this should be a `None`. While `Some(false)` makes a long of sense with the current defaults, it is easier to make changes to defaults and code overall, when there are no surprises between “not provided” and “`-Zplt=no` is provided”.